lib/entity_store_sequel/postgres_entity_store.rb

```diff
@@ -195,18 +195,23 @@ def get_entities(ids, options={})
     def get_events(criteria)
       return {} if criteria.empty?
 
-      result = {}
+      query = events.where('1=2')
```

Comment:
That made me think of a different query to Google and I finally found something: http://sequel.jeremyevans.net/rdoc-plugins/files/lib/sequel/extensions/null_dataset_rb.html